src/makeCalculator.js

```diff
@@ -4,7 +4,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    add: (num) => (result += num),
+
+    subtract: (num) => (result -= num),
+
+    multiply: (num) => (result *= num),
+
+    divide: (num) => (result /= num),
+
+    operate: function (callback, num) {
+      callback(num);
+
+      return this;
+    },
+    reset: function () {
```

Comment:
As per the checklist, you should use the shortcut syntax for defining methods in an object. You can define the 'reset' method as 'reset() { ... }'.